File: AndroidApp.old/RDProject/app/src/main/java/com/rdproj/vli/rdproject/comm/MyoData.java
package com.rdproj.vli.rdproject.comm;

import com.google.gson.annotations.SerializedName;

import org.parceler.Parcel;

@Parcel
public class MyoData {

    @SerializedName("AccX")
    public double accX;
    @SerializedName("AccY")
    public double accY;
    @SerializedName("AccZ")
    public double accZ;

    @SerializedName("GyroX")
    public double gyroX;
    @SerializedName("GyroY")
    public double gyroY;
    @SerializedName("GyroZ")
    public double gyroZ;

    @SerializedName("OriX")
    public double oriX;
    @SerializedName("OriY")
    public double oriY;
    @SerializedName("OriZ")
    public double oriZ;
    @SerializedName("OriW")
    public double oriW;


    @SerializedName("Emg1")
    public int emg1;
    @SerializedName("Emg2")
    public int emg2;
    @SerializedName("Emg3")
    public int emg3;
    @SerializedName("Emg4")
    public int emg4;
    @SerializedName("Emg5")
    public int emg5;
    @SerializedName("Emg6")
    public int emg6;
    @SerializedName("Emg7")
    public int emg7;
    @SerializedName("Emg8")
    public int emg8;

}

File: MyoApp/MyoApp/MyoData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MyoApp
{
    public partial class MyoData
    {
        public double Acc_X { get; set; }

        public double Acc_Y { get; set; }

        public double Acc_Z { get; set; }

        public double Gyro_X { get; set; }

        public double Gyro_Y { get; set; }

        public double Gyro_Z { get; set; }

        public double Ori_X { get; set; }

        public double Ori_Y { get; set; }

        public double Ori_Z { get; set; }

        public double Ori_W { get; set; }

        public double Emg_1 { get; set; }

        public double Emg_2 { get; set; }

        public double Emg_3 { get; set; }

        public double Emg_4 { get; set; }

        public double Emg_5 { get; set; }

        public double Emg_6 { get; set; }

        public double Emg_7 { get; set; }

        public double Emg_8 { get; set; }
        public DateTime Time_stamp { get; set; }

        public double Pitch { get; set; }
        public double Roll { get; set; }
        public double Yaw { get; set; }
        public double EmgMAV { get; set; }

    }
}
